index.js

```diff
@@ -16,82 +15,81 @@ if (process.stdout.isTTY) {
     ns: process.cwd() //A grouping namespace for items.
   });
 
-  const success = chalk.keyword("green");
-  const error = chalk.keyword("red");
-  const warning = chalk.keyword("yellow");
-
-  const { log } = console;
-
   const { loadConfig, execChildProcess } = require("./utils/common");
   const resolveMainTask = require("./utils/resolveMainTask");
   const fetchGitDiff = require("./utils/fetchGitDiff");
-
-  loadConfig()
-    .then(({ config = {} } = {}) => {
-
-      let { base : baseBranch = 'master', tasks = {} } = config;
-      debug('Base Branch:' + baseBranch);
-
-      // Skip linter for base branch
-      let getCurrentBranchCommand = 'git rev-parse --abbrev-ref HEAD';
-      execChildProcess({ command: getCurrentBranchCommand })
-        .then(currentBranch => {
-
-          debug('Current Branch:' + currentBranch);
-
-          if(currentBranch === baseBranch) {
-            log(warning("\nNOTE: Skipping checks since you are in the base branch\n"));
-            return;
-          }
-
-          execChildProcess({ command: 'git rev-parse HEAD' })
-            .then((commitHash = '') => {
-              debug('Curret Commit Hash:' + commitHash);
-
-              let cachedCommitHash = cache.getSync("linted-hash") || "";
-              debug('Cached Commit Hash:' + cachedCommitHash);
-
-              if(commitHash === cachedCommitHash) {
-                log(warning("\nNOTE: Skipping checks since the commit(s) have been linted already.\n"));
-                return;
-              }
-
-              // Fetching committed git files
-              fetchGitDiff( baseBranch ).then((committedGitFiles = []) => {
-                debug(committedGitFiles);
-                new Listr(resolveMainTask({ tasks, committedGitFiles }), {
-                  exitOnError: false,
-                  concurrent: true,
-                  collapse: false
-                })
-                  .run()
-                  .then(() => {
-                    cache.setSync("linted-hash", commitHash);
-                    debug('Cached Current Commit Hash');
-                    log(success("\nVoila! 🎉  Code is ready to be Shipped.\n"));
+  const { theme } = require('./utils/theme');
+
+  theme.then(({ success, error, warning }) => {
```

Comment:
Here the code looks verbose. I have made some changes in master to [load the config file synchronously](https://github.com/theenadayalank/lint-prepush/pull/35/files). Can you push your changes with the recent master?